exercises/getting-started/README.md

```diff
@@ -10,11 +10,7 @@ The goal of this exercise is to:
 
 ## Setup
 
-In your preferred working directory,
-
-- Create a project folder called **`getting-started-with-java`**.
-
-- Inside this project folder:
+- Inside the **`exercises/getting-started`** directory:
```

Comment:
What terminology is used in the slides? Is it directory or folder?

Reply:
Directory

Reply:
folder is used further down in the README